tools/torch/main.lua

```diff
@@ -485,11 +485,13 @@ if opt.optimState ~= '' then
 end
 
 local function updateConfusion(y,yt)
-    if confusion ~= nil then
-        confusion:batchAdd(y,yt)
+    if trainConfusion ~= nil then
+        trainConfusion:batchAdd(y,yt)
     end
 end
 
+local label_function = network.labelHook or function (input, dblabel) return dblabel end
```

Comment:
Indeed the trouble is that the naming conventions are inconsistent in the first place... I was just noting that you seemed to favor camel case notation for globals [there](https://github.com/NVIDIA/DIGITS/pull/495/files#diff-1a7e0f05f8c1adb8faaf9f4f5cda363eR185) and underscore notation for locals [there](https://github.com/NVIDIA/DIGITS/pull/495/files#diff-1a7e0f05f8c1adb8faaf9f4f5cda363eR592). I think that's fine. However [this change](https://github.com/NVIDIA/DIGITS/pull/495/files#diff-1a7e0f05f8c1adb8faaf9f4f5cda363eR493) introduced a new global with underscore notation so I was thinking this wasn't entirely consistent... I am not suggesting that we fix all of these in this pull request but maybe if we could stick to one policy for new code that would be nice.